src/main/java/org/broadinstitute/consent/http/models/DUOSAuthUser.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.consent.http.models;
+
+
+public class DUOSAuthUser extends AuthUser {
```

Comment:
I think it's good that this is a subclass of `AuthUser`, to make migration easier. IMO calling it `DUOSUser` would also make sense, as a DUOS user is a kind of authenticated user, so `DUOSAuthUser` seems redundant.

Also note that the [google java style guidelines](https://google.github.io/styleguide/javaguide.html#s5.3-camel-case) specify that this name should be `DuosAuthUser`. I realize that this isn't how it's been done before, but it's something to consider if we want to adopt those guidelines for DUOS as it's been done for other java terra services.